chi2/chinotype.py

```diff
@@ -141,6 +145,9 @@ def debug_dbopt(self, db):
         log.debug('       chi pats={0}'.format(db['chi_pats']))
         log.debug('    data schema={0}'.format(db['schema']))
         log.debug('data metaschema={0}'.format(db['metaschema']))
+        log.debug('   branch nodes={0}'.format(db['chi_branchnodes']))
+        log.debug('all branch nodes={0}'.format(db['chi_allbranchnodes']))
+        log.debug('     term table={0}'.format(db['chi_termtable']))
```

Comment:
Add the values of the new variables to the log. Say, isn't there some way to just iterate over all the attributes of `self` and log that instead of manually maintaining this list?